### PR TITLE
Show blurred wallpaper when fullscreen

### PR DESCRIPTION
**Pull request by erikinkinen**
Please test this before merging.

### CODE REVIEW

File: src/controls/qml/Wallpaper.qml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    property double maxX: Screen.width*0.1
+    property double maxY: Screen.height*0.1
+
+    ConfigurationValue {
+        id: wallpaperSource
+        key: "/home/glacier/homeScreen/wallpaperImage"
+        defaultValue: "/usr/share/lipstick-glacier-home-qt5/qml/images/wallpaper-portrait-bubbles.png"
+    }
+
+    ConfigurationValue {
```
Comment (neochapay) [inline src/controls/qml/Wallpaper.qml:28]: Not needed here

File: src/styles/qml/SliderStyle.qml [hunk 2]
```diff
@@ -33,7 +33,8 @@ SliderStyle{
         anchors.verticalCenter:useSpecSlider ? undefined : parent.verticalCenter
         y: useSpecSlider ? (control.pressed ? alwaysUp ? parent.y - Theme.itemHeightLarge : ((handleValue  > 0.8) ? parent.y - (Theme.itemHeightLarge*_multiple2) : parent.y ) :  parent.y) : 0
         x: useSpecSlider ? (control.pressed ? alwaysUp ? Theme.itemHeightExtraSmall / 2 : (handleValue > 0.8) ? (_multiple*Theme.itemHeightHuge)  : Theme.itemHeightHuge : 0) : 0
-        color: Theme.backgroundColor
+        //color: Theme.backgroundColor
```
Comment (neochapay) [inline src/styles/qml/SliderStyle.qml]: just remove

File: src/controls/qml/qmldir [hunk 3]
```diff
@@ -33,6 +33,7 @@ TextField 1.0 TextField.qml
 ProgressBar 1.0 ProgressBar.qml
 ActionButton 1.0 ActionButton.qml
 NemoIcon 1.0 NemoIcon.qml
+Wallpaper 1.0 Wallpaper.qml
```
Comment (neochapay) [inline src/controls/qml/qmldir:36]: We don't needed separate controls

File: src/controls/qml/Page.qml [hunk 4]
```diff
@@ -72,7 +72,7 @@ NemoPage {
     Rectangle {
         id: background
```
Comment (neochapay) [inline src/controls/qml/Page.qml]: Why not replace this to blurred image ?

File: src/controls/qml/Wallpaper.qml (hunk 1 above)
Comment (erikinkinen) [reply src/controls/qml/Wallpaper.qml:28]: I believe everything there is needed.

File: src/styles/qml/SliderStyle.qml (hunk 2 above)
Comment (erikinkinen) [reply src/styles/qml/SliderStyle.qml]: Fixed

File: src/controls/qml/Page.qml (hunk 4 above)
Comment (erikinkinen) [reply src/controls/qml/Page.qml]: It is actually easier to remove that item.

File: src/controls/qml/qmldir (hunk 3 above)
Comment (erikinkinen) [reply src/controls/qml/qmldir:36]: Should I embed that into the places where it is used?

Comment (neochapay) [reply src/controls/qml/qmldir:36]: We use Page item. Wallpaper insert of it

File: src/controls/qml/Wallpaper.qml [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+                accelerometer.active = true
+            } else {
+                accelerometer.active = false
+            }
+        }
+    }
+
+    Accelerometer {
+        id: accelerometer
+        active: (enableParallax.value === true && root.size === Screen.size)
```
Comment (neochapay) [inline src/controls/qml/Wallpaper.qml:43]: disable if not used

**Pull request closed without merging**